### ISSUE TITLE
Missing UDP ports opened?

### ISSUE THREAD

**Issue by yspreen** [bredfern/docker-janus#1]

**Comment by bredfern** [comment]
Yeah it needs a turn server I'm working on a docker-compose project for it
that includes nginx and coturn

**Comment by yspreen** [comment]
I'm working on the same thing for Kubernetes. Our coturn is outside k8s so that it can be exposed to the internet 1:1. I'm still facing some connectivity issues though, would you be open to chat about this? Maybe we can exchange some ideas about this!

**Comment by bredfern** [comment]
Sure!

**Comment by bredfern** [comment]
I've had an easy time deploying on lightsail however there load balancing
would be a problem because janus is stateful you'd need a stateful k8s
implementation
